ci/taos/plugins-good/pr-prebuild-cppcheck.sh

```diff
@@ -72,19 +72,20 @@ function pr-prebuild-cppcheck(){
             case $i in
                 # in case of C/C++ code
                 *.c|*.cpp)
+                    # --std=posix is deprecated and removed in 2.05
```

Comment:
Looks good. let's merge this PR. 
It seems that we need to add an exception handling to handle various versions of CPPCheck software.

Reply:
FYI, I uploaded `cppcheck` (2.2.0)  to nnstreamer ppa

Reply:
I updated PR. Please review it again.